crates/criticaltrust/src/revocation_info.rs

```diff
@@ -0,0 +1,47 @@
+// SPDX-FileCopyrightText: The Ferrocene Developers
+// SPDX-License-Identifier: MIT OR Apache-2.0
+
+use crate::keys::KeyRole;
+use crate::signatures::Signable;
+use crate::NoRevocationsCheck;
+use serde::{Deserialize, Serialize};
+use time::OffsetDateTime;
+
+/// Holds hashes of revoked content which are included as a part of the [`KeysManifest`].
+#[derive(Clone, Debug, Serialize, Deserialize)]
+pub struct RevocationInfo {
+    // Incoming SHA256 data from the API is in the form of String, but we save each as a `Vec<u8>`.
+    pub revoked_content_sha256: Vec<Vec<u8>>,
```

Comment:
We should probably make it consistent on both sides.

Reply:
You mean on server side we sent a list of bytes (numbers) right?